app-android-xamarin/Moodis/Moodis/Resources/layout/activity_camera.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout 
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id="@+id/camera_layout">
+    <FrameLayout
```

Comment:
What's the purpose of this frame layout? It is not a good practice to have one layout inside other layout, unless it really necessary.

Reply:
this frame layout contains the camera fragment.

Reply:
Then remove ConstraintLayout and just leave frame layout